src/libraries/System.Data.OleDb/src/System.Data.OleDb.csproj

```diff
@@ -1,15 +1,15 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
-    <TargetFrameworks>netstandard2.0-windows;netstandard2.0;net461-windows</TargetFrameworks>
+    <TargetFrameworks>$(NetCoreAppCurrent)-windows;netstandard2.0-windows;netstandard2.0;net461-windows</TargetFrameworks>
```

Comment:
@ViktorHofer @safern - do we need a `$(NetCoreAppCurrent)` build here as well? i.e.:

```xml
<TargetFrameworks>$(NetCoreAppCurrent)-windows;$(NetCoreAppCurrent);netstandard2.0-windows;netstandard2.0;net461-windows</TargetFrameworks>
```

Reply:
OleDb is taken care of in https://github.com/dotnet/runtime/pull/54544 which I will hopefully merge today.

Reply:
OleDb now has a NetCoreAppCurrent configuration in main.

Reply:
Thanks, I merge changes from main already.